tests/array.rs

```diff
@@ -485,7 +485,7 @@ fn test_add() {
     }
 
     let B = A.clone();
-    A = A + &B;
+    let A = A + &B;
```

Comment:
It's a goal that we can revert this change 🙂

Reply:
That's right. We can achieve it by keeping the limit of DataOwned.